### PR TITLE
Feature/core 1041 show pending invitation

### PR DESCRIPTION
**Pull request by datrinh**
Waiting on https://github.com/evannetwork/ui-dapps/pull/92

### CODE REVIEW

File: dapps/assets/src/i18n/de.json [hunk 1]
```diff
@@ -4,20 +4,20 @@
...
-      "contact-name-placeholder": "Max Mustermann",
-      "contact-name": "Kontaktname",
       "contacts-empty": "Sie haben noch keine Kontakte hinzugefügt.",
       "contacts-title": "Kontakte",
       "created": "Erstellt",
       "delete-modal-title": "Kontakt löschen",
       "edit-contact-btn": "Speichern",
       "edit-contact-title": "Kontakt bearbeiten",
       "email-desc": "Der ausgewählte Nutzer wird mit Ihrer Anfrage kontaktiert. Im folgenden Formular können Sie diese bearbeiten.",
+      "error-added": "The selected account is already in your contact list",
```
Comment (Philser) [inline dapps/assets/src/i18n/de.json]: Not german

File: dapps/assets/src/i18n/de.json [hunk 2]
```diff
@@ -30,6 +30,8 @@
       "message": "Nachricht",
       "name": "Name",
       "note": "Notiz",
+      "note-placeholder": "bspw. ein Alias",
```
Comment (Philser) [inline dapps/assets/src/i18n/de.json]:
```suggestion
      "note-placeholder": "Bspw. ein Alias",
```
Lieber konsistent groß schreiben?

File: dapps/assets/src/i18n/de.json [hunk 3]
```diff
@@ -30,6 +30,8 @@
...
+      "pending": "ausstehend ...",
```
Comment (Philser) [inline dapps/assets/src/i18n/de.json]:
```suggestion
      "pending": "ausstehend ...",
```
Lieber konsistent groß schreiben?

File: dapps/assets/src/i18n/en.json [hunk 4]
```diff
@@ -30,6 +32,8 @@
       "message": "Message",
       "name": "Name",
       "note": "Note",
+      "note-placeholder": "e.g. an alias",
```
Comment (Philser) [inline dapps/assets/src/i18n/en.json]:
```suggestion
      "note-placeholder": "E.g. an alias",
```

File: dapps/assets/src/i18n/en.json [hunk 5]
```diff
@@ -30,6 +32,8 @@
...
+      "pending": "pending ...",
```
Comment (Philser) [inline dapps/assets/src/i18n/en.json]:
```suggestion
      "pending": "Pending ...",
```
s.o.

File: dapps/assets/src/i18n/de.json (hunk 1 above)
Comment (datrinh) [reply dapps/assets/src/i18n/de.json]:
```suggestion
      "error-added": "Der ausgewählte Account ist bereits in Ihrer Kontaktliste.",
```

**Pull request merged**